### PR TITLE
Remove deprecated uses of Error::description.

### PR DESCRIPTION
**Pull request by iancormac84**
Ran tests and it seems to work.

### CODE REVIEW

**Comment by iancormac84** [comment]
Says all checks have failed. Guidance?

**Comment by kvark** [comment]
Travis has been pretty bad lately. I think it wants us to migrate to Travis.com.

**Comment by kvark** [comment]
(Sorry about the delay! Currently swamped with work, barely keeping with notifications)

File: src/de/error.rs [hunk 1]
```diff
@@ -62,7 +62,7 @@ impl fmt::Display for Error {
         match *self {
             Error::IoError(ref s) => write!(f, "{}", s),
             Error::Message(ref s) => write!(f, "{}", s),
-            Error::Parser(_, pos) => write!(f, "{}: {}", pos, self.description()),
+            Error::Parser(_, pos) => write!(f, "{}: {}", pos, self),
```
Comment (CAD97) [inline src/de/error.rs:65]: This is now infinitely recursive!

**Pull request merged**